packages/javalib/javalib.3.0/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+maintainer: "Nicolas Barré <nicolas.barre@irisa.fr>"
+authors: "Javalib Development team"
+homepage: "https://javalib-team.github.io/javalib/"
+bug-reports: "https://github.com/javalib-team/javalib/issues"
+license: "LGPL-2.1 with OCaml linking exception"
+dev-repo: "https://github.com/javalib-team/javalib.git"
+build: [
+  ["./configure.sh"]
+  [make]
+]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "javalib"]
+depends: [
+  "ocamlfind"
```

Comment:
`ocamlfind` seems to only be used during the build phase. Please add the `{build}` attribute